### PR TITLE
Travelling_Salesman.java

### LINKED ISSUE #675: Travelling Salesman Problem (TSP)

### PR DESCRIPTION
**Pull request by mansikagrawal**
Fixes #675

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request: Added file Travelling_Salesman.java
- 
- 

#### Languages Used:Java
- 
- 

#### Files Added:Travelling_Salesman.java
- 
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Travelling_Salesman_Problem/Travelling_Salesman.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
+import java.util.*;
+class TSP 
+{ 
+
+	// Function to find the minimum weight
```
Comment (jainaman224) [inline Travelling_Salesman_Problem/Travelling_Salesman.java]: Use 4 space indentation.

File: Travelling_Salesman_Problem/Travelling_Salesman.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
+import java.util.*;
+class TSP
```
Comment (jainaman224) [inline Travelling_Salesman_Problem/Travelling_Salesman.java]: Add a new line above this.

File: Travelling_Salesman_Problem/Travelling_Salesman.java [hunk 3]
```diff
@@ -0,0 +1,91 @@
+import java.util.*;
+class TSP 
+{ 
+
+	// Function to find the minimum weight 
+	// Hamiltonian Cycle 
+	
+  static int tspfn ( int[][] graph, boolean[] visited, int currPosition, int n, int count, int cost, int answer )
```
Comment (jainaman224) [inline Travelling_Salesman_Problem/Travelling_Salesman.java]: Inconsistent indentation. Please fix throughout the file.

**Comment by mansikagrawal** [comment]
@jainaman224  Please check further changes if required

File: Travelling_Salesman_Problem/Travelling_Salesman.java [hunk 4]
```diff
@@ -0,0 +1,87 @@
+//Travelling Salesman Problem
```
Comment (somya-kapoor) [inline Travelling_Salesman_Problem/Travelling_Salesman.java]: Please use 4 space indentation.

File: Travelling_Salesman_Problem/Travelling_Salesman.java [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+	    int n;
+	    System.out.println(" Enter the number of villages: ");
+	    Scanner in = new Scanner(System.in);
+	    n = in.nextInt();
+		int graph[][] = new int[n][n];
+		System.out.println( "Enter the cost matrix " );
+		for(int i=0;i<n;i++)
+			{
+			    System.out.println( "Enter the cost matrix for village " + i+1 );
+				for(int j=0;j<n;j++)
```
Comment (somya-kapoor) [inline Travelling_Salesman_Problem/Travelling_Salesman.java]: Please add space around the operators.

**Comment by mansikagrawal** [comment]
@somya-kapoor  Mam please check and tell further changes.

**Pull request closed without merging**